e2e/_suites/fleet/features/fleet_server.feature

```diff
@@ -0,0 +1,19 @@
+@fleet_server
+Feature: Fleet Server
+  Scenarios for Fleet Server, where an Elasticseach and a Kibana instances are already provisioned,
+  so that the Agent is able to communicate with them
+
+@start-fleet-server
+Scenario Outline: Deploying the <os> fleet-server agent enables Fleet Server
+  When a "<os>" agent is deployed to Fleet with "tar" installer in fleet-server mode
```

Comment:
@jalvz do you think this step must be rephrased too? IIRC, I decided to create a new one instead of:

```gherkin
When a "<os>" agent is deployed to Fleet with "tar" installer
   And the agent start Fleet Server
```
To avoid invoking the `install` method twice.

Reply:
sgtm!